Content/ReleaseBriefings/4_6/Section_NewFeatures.tex

```diff
@@ -46,5 +46,67 @@
 
 %==========================================================================
 
+\begin{frame}[fragile]\label{sec:new_features}
+
+  {\Huge Kokkos::Graph features}
+
+  \vspace{10pt}
+
+\end{frame}
+
+\begin{frame}[fragile]{Kokkos::Graph recap}
+ \begin{itemize}
+     \item Allows creating a graph of kernels
+     \item Executed in order when calling \texttt{submit(ExecutionSpace)}
+      \begin{code}[keywords={auto}]
+    ...
+    auto graph = Kokkos::Experimental::create_graph(exec, [&](auto root) {
+      auto node_A = root.then_parallel_for(
+                    policy_t(exec, 0, 1),
+                    FetchValuesAndContribute(data, index_A, value_A));
+      auto node_B = root.then_parallel_for(
+                    policy_t(exec, 0, 1),
+                    FetchValuesAndContribute(data, index_B, value_B));}
+    ...
+    graph.submit(exec);
+     \end{code}
+ \end{itemize}
+\end{frame}
+
+\begin{frame}[fragile]{Kokkos::Graph new features}
+ \begin{itemize}
+   \item \texttt{then}-node: adds callable to be executed once
```

Comment:
"to be executed once" might be misleading with the fact that the graph can be submitted many times. I'm not sure my proposal is better, but you get the point :wink:

```tex
\item \href{https://github.com/kokkos/kokkos/pull/7629}{\texttt{then} node}: executes a callable on device
```